packages/block-library/src/video/test/edit.native.js

```diff
@@ -25,5 +25,12 @@ describe( 'Video block', () => {
 		dismissModal( screen.getByTestId( 'bottom-sheet' ) );
 
 		expect( screen.getByText( 'Invalid URL.' ) ).toBeVisible();
+		// Expecations added due to a 17.5.1-specific hotifx: https://github.com/WordPress/gutenberg/pull/58031
```

Comment:
I started by ignoring per test, but the amount of failures in the mobile suites is way too high to add the expectations per test. If the approach below works fine, I'll remove this and use the same env file from other jest.config's.